packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -27,6 +34,13 @@
     <oui-tile-definition
         data-term="{{ ::'vps_dashboard_tile_configuration_memory' | translate }}"
     >
+        <oui-tile-description data-ng-if="$ctrl.statistics">
+            <progress
+                data-ng-class="'oui-progress oui-progress_' + ($ctrl.statistics.memory <= 50 ? 'success' : $ctrl.statistics.memory <= 80 ? 'warning' : 'error')"
```

Comment:
Maybe 50 and 80 could be stored as constants ? It will avoid duplication and can be easily changed if needed

Reply:
done :)